gwpy/table/table.py

```diff
@@ -28,15 +29,14 @@
 
 from gwosc.api import DEFAULT_URL as DEFAULT_GWOSC_URL
 
-from astropy.table import (Table, Column, vstack)
+from astropy.table import (Table, vstack)
 from astropy.io import registry
 
 from ..io.mp import read_multi as io_read_multi
 from ..time import gps_types
 from .filter import (filter_table, parse_operator)
 
 __author__ = 'Duncan Macleod <duncan.macleod@ligo.org>'
-__all__ = ['EventColumn', 'EventTable']
```

Comment:
Should this entire line be deleted or just `'EventColumn', `?  AFAIK, this will only [affect `import *` statements](https://stackoverflow.com/questions/44834/can-someone-explain-all-in-python), but I wanted to make sure the change is intentional.

Reply:
Yes, I intentionally removed the `__all__`, it doesn't really help for a module with such a small set of defined objects.